tests/integration_test/ota_pal/README.md

```diff
@@ -0,0 +1,19 @@
+# OTA PAL Testing
+
+Please provision your device with the correct certificates in `<root-directory>/tests/integration_test/ota_pal/test_files/` before running any OTA PAL tests( these tests are enabled by setting `testrunnerFULL_OTA_PAL_ENABLED` to 1, see `aws_test_runner_config.h` ).
+
+The names of the files correspond to the signing algorithm associated with that certificate.
+
+If your OTA PAL layer port defines `otapalconfigCODE_SIGNING_CERTIFICATE` variable in `ota_pal.c`, then please copy in the certificate, with the applicable signing algorithm for your device, located under `<root-directory>/tests/integration_test/ota_pal/test_files/`. The `otapalconfigCODE_SIGNING_CERTIFICATE` variable is set to use the `<root-directory>/tests/integration_test/ota_pal/test_files/ecdsa-sha256-signer.crt.pem.test` certificate by default.
```

Comment:
I have updated this README.md file to make it more clear in general. I think that the original description does not make as much since after updating the tests. Can you look over this?